Assets/Scripts/CameraFollow.cs

```diff
@@ -43,14 +43,15 @@ private bool CheckYMargin()
 
     private void Update()
     {
-		if (Input.GetKey (KeyCode.C)) {
-			player.GetComponent<Controller2> ().enabled = false;
-			moveCamera ();
-		} else {
-			player.GetComponent<Controller2> ().enabled = true;
-			TrackPlayer();
-		}
-
+		if (!player.GetComponent<Controller2> ().ghostMode) {
+			if (Input.GetKey (KeyCode.C)) {
```

Comment:
We need to changed the Input to handle generic buttons ("Fire1") (look at the Unity input settings). 
This way we can add controller support.